wordpress-mobile/WordPress-iOS
# Add configuration validation during release builds

**To test:**

- Change something in the local secrets repo to cause validation to fail, note that the build fails in release mode, but _not_ in develop mode
